### ISSUE TITLE
Transcription Sheet Set-Up Process

### ISSUE THREAD

**Issue by MasSamH** [codeforamerica/brigade-playbook#59]
### Overview
We need to design a transcription process that covers all interview data and information processing requirements, and recruit and train volunteers in this process to ensure through-flow of transcriptions in support of analysis phase. 

### Action Items

- [ ] Prepare transcription sheets for all completed interviews by
  - [ ] Follow this [procedure (#62)](https://github.com/codeforamerica/brigade-playbook/issues/62) to create individual transcription spreadsheets from template
  - [ ] Place individual transcription spreadsheets into ‘Transcription Storage’ folder (with audio) along with Interview Transcription Procedure document
  - [ ] Add spreadsheet link to interview tracking spreadsheet
- [x] Pilot 
  - [x] Establish what difficulties there could be
  - [x] Revise training documentation
 - [x] ~~Expand pilot to UXR (Tan)~~ Determined automation was more appropriate.
- [x] ~~Recruit and train transcribers~~  Moved to issue #74 
  - Send messaging to recruit transcribers
  - Create Interview Transcription Procedure document (and training slides?)
  - Deliver training to transcribers

### Resources/Instructions
[Transcription Sheet Template](https://docs.google.com/spreadsheets/d/1SDdReZoN20M_uZ5CaN9y-F4mwqZ5pVkIPsA9AlCtjYs/edit#gid=1976949881)
[Interview Transcription Procedure](https://docs.google.com/document/u/1/d/1kTrslqYY0whUjCMxbcV7Ft1nzCr_UTPWXfhomNgOvEw/edit)
[BOP Interview Responses and Tracking](https://docs.google.com/spreadsheets/u/1/d/10uZjjrXpZH-y6wZBHZvhKgWNUBj7Xvyh9yoiqL4F_9I/edit#gid=1930219941)

**Comment by MasSamH** [comment]
@MarianneAnthonette if needed I can split this into separate issues, just seemed a cluster worked, but happy to change.

**Comment by MasSamH** [comment]
Test methodology between Project Team / Jonathan to test [details to be found in #59]
Test one UXR manually with pilot [details to be found in #59]
Rollout training to wider UXR group [details to be found in #59]

**Comment by thadk** [comment]
Mari met with Raquel on past Thursday: added Jon's pieces of procedure to #62.

**Comment by MarianneAnthonette** [comment]
Waiting on Jon and Thad's script delivery on 12/05.

**Comment by MarianneAnthonette** [comment]
Raquel and I completed preparing 24 transcription sheets. @thadk needs to rename the Otter.ai files to match up with the Interview ID (can be found in the [tracking spreadsheet](https://docs.google.com/spreadsheets/d/10uZjjrXpZH-y6wZBHZvhKgWNUBj7Xvyh9yoiqL4F_9I/edit?pli=1#gid=1930219941)).

**Discussion points:**
- Tommy (Open San Diego) and David (Code for Fort Collins): Has the recording been uploaded and where is it?
- Annie (Code for San Jose): How to handle multiple Otter.ai recordings?
- Aaron (KC Digital Drive): Did not make a sheet because the responses need to be redone.
- Laura (Code for Durham): Otter.ai somehow only tracked one speaker.

**Comment by MasSamH** [comment]
Thad has renamed the Otterai files. 

@jkropko @thadk @MarianneAnthonette 

- [ ] Discussion with Jonathan and Thad - ideally Thursday - about how to present more clearly the data from the parsed OtterAI text data tab
- [ ] How we can move the data into the transcription spreadsheets
- [ ] Then we can define the transcription procedure

**Comment by MasSamH** [comment]
@MarianneAnthonette 
- [ ] Marianne defining the steps in the documentation
- [ ] Decide on whether a training session is required
- [ ] Update to bring on Saturday 12.12

**Comment by MarianneAnthonette** [comment]
Made additional changes to the [transcription procedure document](https://docs.google.com/document/d/1kTrslqYY0whUjCMxbcV7Ft1nzCr_UTPWXfhomNgOvEw/edit).

**Comment by celeste-hub** [comment]
@MarianneAnthonette to message @thadk to re-run the script.

**Comment by Baleja** [comment]
@MarianneAnthonette Please review the action items in this issue above. Update according to the new plan.

**Comment by Baleja** [comment]
We're adding this to the review column to make a determination on Saturday on automated or manual transcribing.

**Comment by Baleja** [comment]
@ExperimentsInHonesty to reach out to:
Aaron (KC Digital Drive): Did not make a sheet because the responses need to be redone.
Reach out to Aaron again later 
Gregory (Code for South Florida)

**Comment by Baleja** [comment]
This issue is ongoing as interviews come in.

**Comment by Baleja** [comment]
Closed refer to issue #122 for P&P
